server/reports.go

```diff
@@ -280,21 +281,24 @@ func (server *HTTPServer) getRecommendations(writer http.ResponseWriter, request
 	}
 	log.Info().Int(orgIDStr, int(orgID)).Msg("getRecommendations")
 
-	var listOfClusters []types.ClusterName
+	var listOfClusters []string
```

Comment:
why we changed the type?

Reply:
so that I could use the function created/tested by Bakary `inClauseFromSlice` which requires a slice of strs